### PR TITLE
Address archboard review comments 0804

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationKeyValueData.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public KeyValueData()
         /// A locked key-value may not be modified until it is unlocked.
         /// </param>
         /// <param name="tags"> A dictionary of tags that can help identify what a key-value may be applicable for. </param>
-        internal KeyValueData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, string key, string label, string value, string contentType, string eTag, DateTimeOffset? lastModified, bool? locked, IDictionary<string, string> tags) : base(id, name, resourceType, systemData)
+        internal AppConfigurationKeyValueData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, string key, string label, string value, string contentType, string eTag, DateTimeOffset? lastModified, bool? locked, IDictionary<string, string> tags) : base(id, name, resourceType, systemData)
```
Comment (Yao725) [inline sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationKeyValueData.cs:49]: Should we change the type of `ETag`?

Comment (Yao725) [inline sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationKeyValueData.cs:49]: LastModifiedOn?

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationStoreData.cs [hunk 2]
```diff
@@ -50,7 +50,7 @@ public ConfigurationStoreData(AzureLocation location, AppConfigurationSku sku) :
...
+        internal AppConfigurationStoreData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, IDictionary<string, string> tags, AzureLocation location, ManagedServiceIdentity identity, AppConfigurationSku sku, ProvisioningState? provisioningState, DateTimeOffset? createdOn, string endpoint, Models.EncryptionProperties encryption, IReadOnlyList<PrivateEndpointConnectionReference> privateEndpointConnections, PublicNetworkAccess? publicNetworkAccess, bool? disableLocalAuth, int? softDeleteRetentionInDays, bool? enablePurgeProtection, CreateMode? createMode) : base(id, name, resourceType, systemData, tags, location)
```
Comment (Yao725) [inline sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationStoreData.cs:53]: Maybe we should give `ProvisioningState` and `PublicNetworkAccess` a more specific name.

File: sdk/streamanalytics/Azure.ResourceManager.StreamAnalytics/src/autorest.md [hunk 3]
```diff
@@ -158,6 +177,15 @@ directive:
...
     $.ResourceTestStatus.properties.error['x-ms-client-flatten'] = true;
+- from: outputs.json
+  where: $.definitions
+  transform: >
+    $.PostgreSQLDataSourceProperties.properties.maxWriterCount['type'] = 'integer';
+    $.AzureSqlDatabaseDataSourceProperties.properties.maxWriterCount['type'] = 'integer';
+    $.AzureFunctionOutputDataSourceProperties.properties.maxBatchSize['type'] = 'integer';
+    $.AzureFunctionOutputDataSourceProperties.properties.maxBatchCount['type'] = 'integer';
+    $.AzureSqlDatabaseDataSourceProperties.properties.maxBatchCount['type'] = 'integer';
+    $.ServiceBusQueueOutputDataSourceProperties.properties.systemPropertyColumns['additionalProperties'] = { 'type': 'string' };
```
Comment (Yao725) [inline sdk/streamanalytics/Azure.ResourceManager.StreamAnalytics/src/autorest.md:188]: Why do we need to change this type?

**Review by Yao725 — APPROVED** [review]

File: sdk/streamanalytics/Azure.ResourceManager.StreamAnalytics/src/autorest.md (hunk 3 above)
Comment (ArthurMa1978) [reply sdk/streamanalytics/Azure.ResourceManager.StreamAnalytics/src/autorest.md:188]: From this comment: https://apiview.dev/Assemblies/Conversation/f6ca25aaa01143589fbfdef756473779#482d8b487f7a4ad4a4f0290c8d0bb133

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationKeyValueData.cs (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationKeyValueData.cs]: fixed

Comment (ArthurMa1978) [reply sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationKeyValueData.cs]: fixed

File: sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationStoreData.cs (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Generated/AppConfigurationStoreData.cs]: fixed

**Pull request merged**